src/api-sdks/connection-api/clients/python/ideastatica_connection_api/api_ext/project_ext_api.py

```diff
@@ -1,14 +1,23 @@
 import logging
+import os
+from typing import Optional
 from ideastatica_connection_api.api.project_api import ProjectApi
-from ideastatica_connection_api.api_response import ApiResponse
+from ideastatica_connection_api.models.con_project import ConProject
 
 logger = logging.getLogger(__name__)
 
 class ProjectExtApi(ProjectApi):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.active_project_data: Optional[ConProject] = None
         # Add any additional initialization here
 
+    @property
+    def project_id(self):
```

Comment:
@OndraZitko I would call this property active_project_id and the client should have the same property which gets it from the project extension.